.github/workflows/bot-mere-denis.yml

```diff
@@ -0,0 +1,51 @@
+name: "[Bot] 'Mère Denis'"
+on:
+  push:
+    branches:
+      - support/bot-mere-denis
+      - support/bot-actions-perf
+
+concurrency:
+  group: bot-seed1
+
+jobs:
+  start-runner:
+    name: "start ec2 instance (Linux)"
+    if: ${{ always() }}
```

Comment:
Just for my culture, no idea how this works but are the conditions a must have ? What is this always() thingie ? Thanks :)

Reply:
i'm not familiar with the need in this specific ec2 instance. ask live-dx team 🙏